interface/wx/bitmap.h

```diff
@@ -762,6 +767,31 @@ class wxBitmap : public wxGDIObject
             Bitmap width in pixels.
     */
     virtual void SetWidth(int width);
+
+    /**
+       Retrieves the size of a device-independent standard pixel relative to that
+       of a device-dependent, actual pixel for a scaled bitmap
+       @since 3.1.5
```

Comment:
3.1.5 doesn't seem correct here. These functions were added to the base bitmap class in 3.0 (albeit, the GTK port didn't get native implementations until 3.1.0, and I can't tell if MSW has a native implementation or if it needs it).